web/modules/custom/joinup_news/joinup_news.module

```diff
@@ -53,14 +52,7 @@ function joinup_news_workflow_selector(EntityInterface $entity) {
   }
   /** @var \Drupal\joinup_core\JoinupRelationManager $relation_manager */
   $relation_manager = \Drupal::service('joinup_core.relations_manager');
-  try {
-    $moderation = $relation_manager->getParentModeration($entity);
-  }
-  catch (MissingRelationException $e) {
-    // Throw a more helpful exception if the news article doesn't have a parent
-    // collection or solution.
-    throw new \Exception('Cannot determine the moderation workflow type on a news article that doesn\'t belong to a collection or solution.');
-  }
+  $moderation = $relation_manager->getParentModeration($entity);
```

Comment:
Why remove this? I added this exception specifically because I was getting a nondescript error before (like 'accessing propery on non-object') which cost me time to figure out.

This problem can occur very easily, if you just create any community content in a Behat test without associating it with a collection or solution. Now at least we get a clear exception instead of having to hunt deep into a stack trace.